### PR TITLE
Two homography features

### PR DESCRIPTION
**Pull request by saching13**
added calibration api along with rectified left and rectified right stream information to the documentation.

### CODE REVIEW

File: api.md [hunk 1]
```diff
@@ -293,6 +295,41 @@ pipeline = device.create_pipeline(config={
...
+
+{: #get_translation}
+* [__get_translation__](#get_translation)()
+    
+    Return a 3x1 vector repesenting the position of the right stereo camera center w.r.t left stereo camera center.
+<div class="alert alert-primary" role="alert">
+<i class="material-icons">
+Note:
+</i>
+  if you have been using old build system and recently moved to the new build system then to enable `get_left_homography()`, `get_left_intrinsic()`, `get_right_intrinsic()`, `get_rotation()`, `get_translation()` you need to recalibrate your device by following the tutorial <strong><a href="https://docs.luxonis.com/tutorials/stereo_calibration/">here</a></strong><br/> However, get_right_homography() works even if you were using the old build without any extra effort.
```
Comment (Luxonis-Brandon) [inline api.md:331]: This renders funny:
image

I think from here it shouldn't have `Note` in there but instead `error` and be like below:

```
<div class="alert alert-primary" role="alert">
<i class="material-icons">
error
</i>
  Using the RPi Compute Edition ([BW1097](https://docs.luxonis.com/products/bw1097/)) or USB3 Onboard Cameras Edition ([BW1098OBC](https://docs.luxonis.com/products/bw1098obc/))? <strong>Your unit comes pre-calibrated.</strong><br/>
</div>
```

From the top of here:
https://github.com/luxonis/depthai-docs-website/blob/master/_includes/stereo_calibration.md

**Review by Luxonis-Brandon — COMMENTED** [review]
I'm going to try changing that formatting and a bit of the wording in there.

File: api.md (hunk 1 above)
Comment (saching13) [reply api.md:331]: That is weird. WIll change it.

**Comment by VanDavv** [comment]
@saching13 I can see you're operating on an outdated branch (and some of the changes overlap, like available stream names). Could you rebase / merge with latest master?
image

**Review by VanDavv — CHANGES_REQUESTED** [review]
@saching13 I can see you're operating on an outdated branch (and some of the changes overlap, like available stream names). Could you rebase / merge with latest master?
image

**Comment by saching13** [comment]
@VanDavv Done.

**Comment by Luxonis-Brandon** [comment]
Sorry this one was my fault I think.

**Pull request merged**